src/styles/blocks/reset.scss

```
@@ -0,0 +1,19 @@
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+h2 {
+  margin: 0;
+  padding: 0;
+}
+
+p {
```

h2 & p has a default padding?))
```suggestion
h2, 
p {
```